build-aux/main.mk

```diff
@@ -12,15 +12,15 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
```

Comment:
I know this is the "main" file, but like you've added in the other *.mk files, I think a blurb about the purpose of this file would be helpful.. something like: "This file deals with the linting, testing, building, publishing, and releasing of Telepresence artifacts". 

 I think it's good to have a specified description for each file so if some new _thing_ comes along we know whether it makes sense to put it in here or a different *.mk file.. this will also prevent main.mk from being a catch-all location.